app/javascript/packs/recommend_pages.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { render } from 'react-dom';
+import ComponentWrapper from '../components/ComponentWrapper';
+
+const store = window.champaign.store;
+
+class RecommendPagesView extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      pages: [],
+    };
+  }
+
+  render() {
+    return (
+      <div className="campaign-tiles">
+        {this.state.pages.map(function(page) {
+          return <CampaignTile key={page.id} page={page} />;
+        })}
```

Comment:
This could be expressed on a single line:
```javascript
{this.state.pages.map( page =>  <CampaignTile key={page.id} page={page} /> )}
```

Reply:
you could also pass the page properties directly `<CampaignTile key={page.id} {...page} />`, allowing you to access them directly from `props` in the campaign tile component.

Reply:
Many thanks for lending me your keen eye. I've implemented the requested changes.